src/NUnitConsole/nunit-console/ConsoleRunner.cs

```diff
@@ -185,11 +186,34 @@ private int RunTests(TestPackage package, TestFilter filter)
         private void WriteRuntimeEnvironment(ExtendedTextWriter OutWriter)
         {
             OutWriter.WriteLine(ColorStyle.SectionHeader, "Runtime Environment");
-            OutWriter.WriteLabelLine("   OS Version: ", Environment.OSVersion.ToString());
+            OutWriter.WriteLabelLine("   OS Version: ", GetOSVersion());
             OutWriter.WriteLabelLine("  CLR Version: ", Environment.Version.ToString());
             OutWriter.WriteLine();
         }
 
+        private static string GetOSVersion()
+        {
+            OperatingSystem os = Environment.OSVersion;
+            string osString = os.ToString();
+            if (os.Platform == PlatformID.Unix && os.Platform != PlatformID.MacOSX)
```

Comment:
This if doesn't make a lot of sense. It seems that the second compare has to succeed if the first one succeeds. If there is some magic here, please add a comment. :-)